src/lasp_core.erl

```diff
@@ -914,7 +914,7 @@ increment_counter(Counter) ->
 
 %% @private
 store_delta(Type, Counter, Delta, DeltaMap0) ->
-    MaxDeltaSlots = application:get_env(lasp, delta_mode_max_slots, 10),
+    MaxDeltaSlots = mochiglobal:get(delta_mode_max_slots, 10),
```

Comment:
You also need to set it in the `lasp_sup` file, else it will always be undefined.